test-two/src/index.js

```diff
@@ -3,21 +3,36 @@ import { getUsersForOrganisation } from "./services/github";
 
 const baseUrl = "https://api.github.com/";
 const organisationId = "google";
-const gitHubApiAuthToken = ""; // add your GitHub API OAuth key here to increase request limit
+const gitHubApiAuthToken = "";
+export const pages = 22; // There are 2144 users so 22 pages are sufficient
 
 export const go = async () => {
-  try {
+  let output = "";
+  let parallellCalls = [];
+
+  const getPageOfUsers = async pageNumber => {
     const users = await getUsersForOrganisation(
       baseUrl,
       gitHubApiAuthToken,
-      organisationId
+      organisationId,
+      pageNumber
     );
 
-    const output = users.reduce((result, user) => {
+    output += users.reduce((result, user) => {
       return `${result}Username: ${user.login}\n`;
     }, "");
```

Comment:
With all the parallelCalls' async functions resolving at different times, would this guarantee the same order of results each time? Is there a better way to order the results?

Reply:
You're right, the results would be added to `output` whenever the individual functions resolve.

At this time, the way I've set it up here is the fastest way I could think of to get the results. The downside is that the final output wouldn't be sorted.

One way I can think of to ensure that the results are sorted would be to have the functions run in order but that sacrifices performance significantly.

Another (slightly convoluted) way would be to first store the results in an array of strings, then sort the array and convert it into a string with all the users properly sorted.